mycroft/skills/intent_service.py

```diff
@@ -316,7 +316,11 @@ def handle_utterance(self, message):
             # Get language of the utterance
             lang = message.data.get('lang', "en-us")
             set_active_lang(lang)
-            utterances = message.data.get('utterances', '')
+
+            # normalize() changes "it's a boy" to "it is a boy", etc.
+            utterances = [normalize(u.lower(), remove_articles=False)
+                          for u in message.data.get('utterances', '')]
```

Comment:
This one should for consistency default to the empty list `[]`

Reply:
Yup!